api/src/org/apache/cloudstack/api/response/TemplateResponse.java

```diff
@@ -169,6 +169,10 @@
     @Param(description = "additional key/value details tied with template")
     private Map details;
 
+    @SerializedName(ApiConstants.BITS)
+    @Param(description="the processor bit size")
+    private int bits;
```

Comment:
Why is this value represented as a number and not an enumeration?  It can't be any number other 32 and 64.